jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -711,6 +731,28 @@ def getLambdaARN (String stackName) {
 
 	return ARN;
 }
+
+/** Run validation based on runtime
+*/
+def runValidation(String runtime) {
+	if(runtime.indexOf("nodejs") > -1) {
+		echo "running validations for $runtime"
+		sh "jshint *.js"
+	}else if(runtime.indexOf("java") > -1) {
```

Comment:
indentation here and other places in the code

Reply:
done